docs/framework/guides/support/getting-support.md

```diff
@@ -41,3 +41,9 @@ DON'Ts:
 * The editor crashes if I press Enter. Please help.
 
 And most importantly, be kind to everyone.
+
+## Commercial support
+
+CKEditor 5 also comes with commercial licensing benefits, such as direct support from the core CKEditor development team. Commercial licenses are designed with professionals and enterprises in mind.
+
+For more information, check [the CKEditor 5 Pricing page](https://ckeditor.com/ckeditor-5-builds/pricing/).
```

Comment:
Should "the" be linked?

Reply:
I don't know - you tell me :)

Reply:
I've never link it so let's be consistent. But I never checked how people do this.